_test/test/goldens/generated_build_script.dart

```diff
@@ -138,8 +139,8 @@ final _builders = <_i1.BuilderApplication>[
   _i1.applyPostProcess(
       r'provides_builder:some_post_process_builder', _i4.somePostProcessBuilder)
 ];
-void main(List<String> args, [_i9.SendPort sendPort]) async {
-  var result = await _i10.run(args, _builders);
+_i9.Future<void> main(List<String> args, [_i10.SendPort sendPort]) async {
```

Comment:
Why did this return type need to change?

Reply:
I thought it might be more appropriate given the `async` modifier, but now that we have full customization over the build script, that doesn't matter much to me anymore. I've reverted the change.